install.sh

```diff
@@ -93,6 +93,35 @@ required() {
   fi
 }
 
+display_usage() {
+  echo "ockam/develop/install.sh - attempt to install the ockam binary"
+  echo " "
+  echo "Usage: install.sh [OPTION]..."
+  echo "    -h, --help                show brief help"
+  echo "    -p, --install-path PATH   specify the location for installation"
+  echo "                              (default path is ~/.ockam)"
+  echo "    -v, --version VERSION     specify the version to install"
+  echo "        --no-modify-path      do not add ockam to the PATH"
+}
+
+# using custom function instead of eval due to associated risks of using
+# eval to expand user inputted filepath
+expand_filepath() {
+  local _path="$1"
+  _path="${_path/#\./$(pwd)}"
```

Comment:
@quasiuslikecautious This is looking pretty close to ready except for on minor issue.
shfmt fails on this line because we're expecting any posix shell and search and replace is a bash/mksh feature
https://github.com/build-trust/ockam/actions/runs/4448885465/jobs/7812303056?pr=4480

P.S I force pushed to your branch to tackle some minor lints, mainly that we use spaces for indenting scrips.

Reply:
Just pushed latest changes in new commit (and also resolved merge conflict)  - switched expand_filepath to use sed + regex to replace filepath symbols instead of bash command. Let me know if any other changes are needed, and thanks again for the feedback/reviewing!